src/signals/incident/containers/IncidentContainer/reducer.test.js

```diff
@@ -197,10 +204,22 @@ describe('signals/incident/containers/IncidentContainer/reducer', () => {
             handling_message,
           },
           loadingClassification: false,
+          loadingQuestions: false,
           classificationPrediction: classification,
         })
       })
 
+      it('sets loadingQuestions with feature flag enabled', () => {
```

Comment:
We can see that the feature flag is enabled. What does it mean? That it is specified that questions are retrieved from the backend?